packages/monaco/src/browser/monaco-editor-service.ts

```diff
@@ -66,6 +66,10 @@ export class MonacoEditorService extends monaco.services.CodeEditorServiceImpl {
         return editor && editor.getControl();
     }
 
+    getFocusedCodeEditor(): monaco.editor.IStandaloneCodeEditor | undefined {
```

Comment:
I'm not sure I like this change: the active editor and the focused editor are not the same concept. We don't know what this might break in the VS Code code we're consuming if we make them the same.

Reply:
We agree that this isn't optimal solution but we haven't found a better one currently.

On VS Code side it is checking only focusedCodeEditor
![image](https://user-images.githubusercontent.com/71069170/122780179-48f81c80-d2b7-11eb-8d40-57a205f0c038.png)

While on Theia' side we are checking both focused and active before running action:
![image](https://user-images.githubusercontent.com/71069170/122780311-6af19f00-d2b7-11eb-850e-32ce7f7d8a01.png)